src/NHSD.BuyingCatalogue.Ordering.Api/Models/Summary/OrderSummaryModel.cs

```diff
@@ -25,7 +25,7 @@ public static OrderSummaryModel Create(Order order)
             if (order is null)
                 throw new ArgumentNullException(nameof(order));
 
-            int serviceRecipientsCount = order.SelectedServiceRecipients.Count;
+            int serviceRecipientsCount = order.OrderItems.Where(o => o.CatalogueItem.CatalogueItemType == CatalogueItemType.Solution).SelectMany(o => o.OrderItemRecipients).Count();
```

Comment:
I think we can remove `serviceRecipientsCount` and the `SectionModel.ServiceRecipients` below (line 47).

Reply:
Removed `serviceRecipientsCount` with PR [435](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueOrdering/pull/435)